docs/updates/2019-04.md

```diff
@@ -0,0 +1,180 @@
+
+App Analysis Commands:
+  assoc         Print table associations
+  catwalk       Open the specified app in catwalk
+  eval          Evaluate a list of measures and dimensions
+  fields        Print field list
+  keys          Print key-only field list
+  meta          Print tables, fields and associations
+  tables        Print tables
+  values        Print the top values of a field
+
+Advanced Commands:
+  app           Explore and manage apps
+  connection    Explore and manage connections
+  dimension     Explore and manage dimensions
+  measure       Explore and manage measures
+  object        Explore and manage generic objects
+  script        Explore and manage the script
+
+Other Commands:
+  completion    Generate auto completion scripts
+  help          Help about any command
+  status        Print status info about the connection to the engine and current app
+  version       Print the version of corectl
+
+Flags:
+  -a, --app string               App name, if no app is specified a session app is used instead
+  -c, --config string            path/to/config.yml where parameters can be set instead of on the command line
+  -e, --engine string            URL to the Qlik Associative Engine (default "localhost:9076")
+      --headers stringToString   Http headers to use when connecting to Qlik Associative Engine (default [])
+  -h, --help                     help for corectl
+      --no-data                  Open app without data
+  -t, --traffic                  Log JSON websocket traffic to stdout
+      --ttl string               Qlik Associative Engine session time to live in seconds (default "30")
+  -v, --verbose                  Log extra information
+```
+
+Among all the changes the following are noteworthy:
+
+### Human readable vs json output
+
+Since we assume corectl will be used interactively and in scripts we default to a human readable table while giving an
+option to go for json output for scripting purposes.
+
+```bash
+corectl -a gdq.qvf measure ls
++-----------+-------+
+|    Id     | Title |
++-----------+-------+
+| measure-1 |       |
++-----------+-------+
+
+vs
+corectl -a gdq.qvf measure ls --json
+[
+  {
+    "qId": "measure-1",
+    "title": ""
+  }
+]
+```
+
+### Support for ABAC enabled applications
+
+### Support environment variables in config file
+
+### Other improvements
+
+Apart from all the larger improvements we have also delivered:
+
+* Open app without data (using the --no-data flag)
+* Add bash completion for rm commands
+* Enforce number of parameters for each command
+* Improved config file by givin informative error messages
+* Handle objects without data, empty prints and entities which can't be found
+* add suggestion on commands on erroneous input
+* add --no-reload flag to avoid reloading during build
+
+## hamus.js released as npm package
+
+The hamus.js project has been released as a [npm package](https://www.npmjs.com/package/hamus.js) for easy integration
+in new projects.
```

Comment:
Maybe link to catwalk implementation if they want more real-world examples?

Reply:
It is not used in catwalk as far as I know yet.

Reply:
My bad, thought Helene showed that to me the other day 🤔